{{cookiecutter.project_slug}}/setup.py

```diff
@@ -23,20 +23,20 @@
 
 # Requirements for {{cookiecutter.module_name}}
 requirements = [
-    "OpenEye-orionplatform==1.0.0",
-    "OpenEye-snowball==0.16.0",
```

Comment:
@mattgeballe This is not available to customers it seems. Can't use orion platform 1.0.0 until Snowball 0.16.0. Should we roll the back to a version of snowball that is publicly available?

Reply:
Found this by testing this on orion.eyesopen.com

Reply:
Probably, at least in the short term until packages go public with the 2019.4 release. looks like snowball v0.14.0 is the most recent public version.